powerdns/tests/test_api.py

```diff
@@ -0,0 +1,72 @@
+# -*- encoding: utf-8 -*-
+
+from urllib.parse import urlencode
+
+from django.contrib.auth import get_user_model
+from django.http import QueryDict
+from django.test import TestCase
+from rest_framework.test import APIClient, APIRequestFactory
+
+from powerdns.tests.utils import DomainFactory
+from powerdns.tests.utils import RecordFactory
+from powerdns.utils import AutoPtrOptions
+from powerdns.views import RecordViewSet
+
+
+class TestApi(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.request_factory = APIRequestFactory()
+        get_user_model().objects.create_superuser(
+            'test', 'test@test.test', 'test'
+        )
+        self.client = APIClient()
+        self.client.login(username='test', password='test')
+
+        domain1 = DomainFactory(
+            name='example.com', template=None, reverse_template=None)
+        self.a_record1 = RecordFactory(
+            domain=domain1, type='A', name='www.example.com',
+            content='192.168.1.1', auto_ptr=AutoPtrOptions.NEVER,
```

Comment:
why never? imo we should create ptr here and check if ptr record is in result later